modules/apps/commerce/commerce-cart-content-web/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"dependencies": {
+		"frontend-js-web": "*"
+	},
+	"name": "@liferay/commerce-cart-content-web",
+	"scripts": {
+		"build": "liferay-npm-scripts build",
+		"checkFormat": "liferay-npm-scripts check",
+		"format": "liferay-npm-scripts fix"
+	},
+	"version": "4.0.38"
```

Comment:
@brianikim @kresimir-coko still I don't get why this is not something like 1.0.0

Reply:
If in doubt, ask in a public channel. I've got no definitive answer here.